utils/window/window_test.go

```diff
@@ -103,20 +103,32 @@ func TestTTLAdd(t *testing.T) {
 	oldest, ok = window.Oldest()
 	require.True(ok)
 	require.Equal(4, oldest)
+	// Now we're one second before the ttl of 10 seconds of when [4] was added,
+	// no element should be evicted
+	// [4, 5]
+	clock.Set(epochStart.Add(20 * time.Second))
+	window.Add(5)
+	require.Equal(2, window.Length())
+	oldest, ok = window.Oldest()
+	require.True(ok)
+	require.Equal(4, oldest)
+
 	// Now we're one second past the ttl of 10 seconds of when [4] was added,
 	// so all existing elements should be evicted.
 	clock.Set(epochStart.Add(22 * time.Second))
```

Comment:
This statement doesn't seem to do anything anymore.

Reply:
removed